### ISSUE TITLE
Required Pretrained Model vit_base_p16_224.pth

### ISSUE THREAD

**Issue by t-rizvi** [XinyuSun/SVFormer#1]
Hi 

Can you share the pre-trained model used in this repository ?

**Comment by t-rizvi** [comment]
I trained a pretrained network from internet. Partial Log can be find below that seems incorrect as error values are approaching zero :

[05/26 13:40:38][INFO] helpers.py:  48: Loaded state_dict from checkpoint 'output/vit_base_p16_224.pth'
[05/26 13:40:38][INFO] helpers.py:  48: Loaded state_dict from checkpoint 'output/vit_base_p16_224.pth'
[05/26 13:40:40][INFO] unlabel.py: 121: Constructing Kinetics train...
[05/26 13:40:40][INFO] unlabel.py: 161: Constructing kinetics dataloader (size: 101) from ./dataset/list_ucf_1/train.csv
[05/26 13:40:40][INFO] unlabel.py: 121: Constructing Kinetics val...
[05/26 13:40:40][INFO] unlabel.py: 161: Constructing kinetics dataloader (size: 3783) from ./dataset/list_ucf_1/val.csv
[05/26 13:40:40][INFO] unlabel.py: 121: Constructing Kinetics unlabel...
[05/26 13:40:40][INFO] unlabel.py: 161: Constructing kinetics dataloader (size: 9436) from ./dataset/list_ucf_1/unlabel.csv
[05/26 13:40:40][INFO] tensorboard_vis.py:  56: To see logged results in Tensorboard, please launch using the command             `tensorboard  --port=<port-number> --logdir ./output/list_ucf_1/ucf101_1_reproduce/runs-Unlabel`
[05/26 13:40:40][INFO] train_net_emamix.py: 724: Start epoch: 1
[05/26 13:41:35][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.80039, "dt_data": 0.00064, "dt_net": 1.79975, "epoch": "1/30", "eta": "9:25:55", "gpu_mem": "22.44G", "iter": "10/629", "loss": 6.22161, "loss_label": 6.22161, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 100.00000, "top5_err": 100.00000}
[05/26 13:41:49][INFO] logging.py:  95: json_stats: {"RAM": "134.31/251.79G", "_type": "train_epoch", "dt": 0.82920, "dt_data": 0.82920, "dt_net": 1.80681, "epoch": "1/30", "eta": "4:12:04", "gpu_mem": "22.44G", "loss": 6.15252, "lr": 0.00500, "top1_err": 100.00000, "top5_err": 98.03922}
[05/26 13:42:45][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.80598, "dt_data": 0.00049, "dt_net": 1.80549, "epoch": "2/30", "eta": "9:08:44", "gpu_mem": "22.44G", "iter": "10/629", "loss": 6.04625, "loss_label": 6.04625, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 100.00000, "top5_err": 100.00000}
[05/26 13:42:59][INFO] logging.py:  95: json_stats: {"RAM": "134.36/251.79G", "_type": "train_epoch", "dt": 0.91609, "dt_data": 0.91609, "dt_net": 1.81126, "epoch": "2/30", "eta": "4:28:52", "gpu_mem": "22.44G", "loss": 5.97695, "lr": 0.00500, "top1_err": 98.03922, "top5_err": 95.09804}
[05/26 13:43:55][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.80895, "dt_data": 0.00058, "dt_net": 1.80837, "epoch": "3/30", "eta": "8:50:41", "gpu_mem": "22.44G", "iter": "10/629", "loss": 5.63177, "loss_label": 5.63177, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 100.00000, "top5_err": 83.33334}
[05/26 13:44:08][INFO] logging.py:  95: json_stats: {"RAM": "134.56/251.79G", "_type": "train_epoch", "dt": 0.84231, "dt_data": 0.84231, "dt_net": 1.81708, "epoch": "3/30", "eta": "3:58:24", "gpu_mem": "22.44G", "loss": 5.60655, "lr": 0.00500, "top1_err": 96.07843, "top5_err": 87.25490}
[05/26 13:45:04][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.81344, "dt_data": 0.00055, "dt_net": 1.81289, "epoch": "4/30", "eta": "8:32:59", "gpu_mem": "22.44G", "iter": "10/629", "loss": 5.21229, "loss_label": 5.21229, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 100.00000, "top5_err": 66.66667}
[05/26 13:45:18][INFO] logging.py:  95: json_stats: {"RAM": "134.62/251.79G", "_type": "train_epoch", "dt": 0.85879, "dt_data": 0.85879, "dt_net": 1.89517, "epoch": "4/30", "eta": "3:54:03", "gpu_mem": "22.44G", "loss": 5.20921, "lr": 0.00500, "top1_err": 90.19608, "top5_err": 67.64706}
[05/26 13:46:14][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.81788, "dt_data": 0.00054, "dt_net": 1.81734, "epoch": "5/30", "eta": "8:15:11", "gpu_mem": "22.44G", "iter": "10/629", "loss": 4.84387, "loss_label": 4.84387, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 66.66667, "top5_err": 58.33334}
[05/26 13:46:28][INFO] logging.py:  95: json_stats: {"RAM": "134.72/251.79G", "_type": "train_epoch", "dt": 0.82352, "dt_data": 0.82350, "dt_net": 1.96701, "epoch": "5/30", "eta": "3:35:48", "gpu_mem": "22.44G", "loss": 4.75532, "lr": 0.00500, "top1_err": 66.66667, "top5_err": 48.03922}
[05/26 13:47:20][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:02:46", "gpu_mem": "22.44G", "iter": "10/61", "time_diff": 3.25905, "top1_err": 90.47620, "top5_err": 71.42857}
[05/26 13:47:31][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:00:39", "gpu_mem": "22.44G", "iter": "20/61", "time_diff": 0.96735, "top1_err": 88.88889, "top5_err": 74.60318}
[05/26 13:48:02][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:00:26", "gpu_mem": "22.44G", "iter": "30/61", "time_diff": 0.86522, "top1_err": 87.30159, "top5_err": 73.80952}
[05/26 13:48:21][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:00:19", "gpu_mem": "22.44G", "iter": "40/61", "time_diff": 0.90611, "top1_err": 87.30159, "top5_err": 74.60318}
[05/26 13:48:38][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:01:31", "gpu_mem": "22.44G", "iter": "50/61", "time_diff": 8.27954, "top1_err": 88.09524, "top5_err": 74.60318}
[05/26 13:48:47][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "5/30", "eta": "0:00:00", "gpu_mem": "22.44G", "iter": "60/61", "time_diff": 0.87057, "top1_err": 91.26984, "top5_err": 73.80953}
[05/26 13:48:48][INFO] logging.py:  95: json_stats: {"RAM": "127.42/251.79G", "_type": "val_epoch", "epoch": "5/30", "gpu_mem": "22.44G", "min_top1_err": 89.33126, "min_top5_err": 74.49909, "time_diff": 1.37543, "top1_err": 89.33126, "top5_err": 74.49909}
[05/26 13:49:43][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.81533, "dt_data": 0.00069, "dt_net": 1.81464, "epoch": "6/30", "eta": "7:55:27", "gpu_mem": "22.44G", "iter": "10/629", "loss": 4.11215, "loss_label": 4.11215, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 33.33334, "top5_err": 16.66667}
[05/26 13:49:57][INFO] logging.py:  95: json_stats: {"RAM": "142.77/251.79G", "_type": "train_epoch", "dt": 0.84859, "dt_data": 0.84859, "dt_net": 1.82359, "epoch": "6/30", "eta": "3:33:29", "gpu_mem": "22.44G", "loss": 4.15858, "lr": 0.00500, "top1_err": 43.13726, "top5_err": 18.62745}
[05/26 13:50:53][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.81751, "dt_data": 0.00057, "dt_net": 1.81694, "epoch": "7/30", "eta": "7:36:59", "gpu_mem": "22.44G", "iter": "10/629", "loss": 3.70008, "loss_label": 3.70008, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 16.66667, "top5_err": 0.00000}
[05/26 13:51:07][INFO] logging.py:  95: json_stats: {"RAM": "142.78/251.79G", "_type": "train_epoch", "dt": 0.81796, "dt_data": 0.81796, "dt_net": 1.88388, "epoch": "7/30", "eta": "3:17:12", "gpu_mem": "22.44G", "loss": 3.58455, "lr": 0.00500, "top1_err": 15.68628, "top5_err": 4.90196}
[05/26 13:52:03][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.82087, "dt_data": 0.00099, "dt_net": 1.81988, "epoch": "8/30", "eta": "7:18:44", "gpu_mem": "22.44G", "iter": "10/629", "loss": 3.03203, "loss_label": 3.03203, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 0.00000, "top5_err": 0.00000}
[05/26 13:52:17][INFO] logging.py:  95: json_stats: {"RAM": "144.95/251.79G", "_type": "train_epoch", "dt": 0.73500, "dt_data": 0.73501, "dt_net": 1.94983, "epoch": "8/30", "eta": "2:49:30", "gpu_mem": "22.44G", "loss": 3.07809, "lr": 0.00500, "top1_err": 8.82353, "top5_err": 4.90196}
[05/26 13:53:13][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.81684, "dt_data": 0.00054, "dt_net": 1.81630, "epoch": "9/30", "eta": "6:58:43", "gpu_mem": "22.44G", "iter": "10/629", "loss": 2.61256, "loss_label": 2.61256, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 0.00000, "top5_err": 0.00000}
[05/26 13:53:28][INFO] logging.py:  95: json_stats: {"RAM": "143.16/251.79G", "_type": "train_epoch", "dt": 0.81191, "dt_data": 0.81191, "dt_net": 3.19927, "epoch": "9/30", "eta": "2:58:43", "gpu_mem": "22.44G", "loss": 2.66723, "lr": 0.00500, "top1_err": 7.84314, "top5_err": 0.98039}
[05/26 13:54:24][INFO] logging.py:  95: json_stats: {"_type": "train_iter", "dt": 1.86806, "dt_data": 0.00056, "dt_net": 1.86750, "epoch": "10/30", "eta": "6:50:56", "gpu_mem": "22.44G", "iter": "10/629", "loss": 2.23044, "loss_label": 2.23044, "loss_unlabel": 0.00000, "lr": 0.00500, "top1_err": 0.00000, "top5_err": 0.00000}
[05/26 13:54:39][INFO] logging.py:  95: json_stats: {"RAM": "142.99/251.79G", "_type": "train_epoch", "dt": 0.79896, "dt_data": 0.79896, "dt_net": 2.16050, "epoch": "10/30", "eta": "2:47:30", "gpu_mem": "22.44G", "loss": 2.16795, "lr": 0.00500, "top1_err": 0.98039, "top5_err": 0.00000}
[05/26 13:55:54][INFO] logging.py:  95: json_stats: {"_type": "val_iter", "epoch": "10/30", "eta": "0:00:47", "gpu_mem": "22.44G", "iter": "10/61", "time_diff": 0.92307, "top1_err": 57.93650, "top5_err": 36.50794}

.
.
.
